maap/maap.py

```diff
@@ -256,13 +256,76 @@ def cancelJob(self, jobid):
         job.id = jobid
         return job.cancel_job()
 
-    def listJobs(self, username=None, page_size=None, offset=None):
-        if username==None and self.profile is not None and 'username' in self.profile.account_info().keys():
+    def listJobs(self, username=None, 
```

Comment:
Given the large number of options, use keyword-only parameters. Specifically, inserting the `*` forces the caller to use keywords for any parameters listed _after_ the `*`.

```suggestion
    def listJobs(self, *, username=None, 
```

This means you must call the function like so: `listJobs(username=username, algo_id="myalgo", ...)`

Otherwise, it can be very confusing with so many args if you _don't_ force use of keywords.

Reply:
This would be a breaking change in the interface, which I do not want to do without first communicating the change to users. I made all parameters other than `username` keyword-only params.

More broadly speaking, I believe we have plans to cleanup maap-py in the nearish future and can revisit this then.